CodeFileSanity/CodeFileSanity.csproj

```diff
@@ -1,12 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup Label="Project">
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>

```

Comment:
is the net46 necessary? feels like a step backwards.

Reply:
Sadly addins require this framework combination. Cake automaticly targets net46 on windows and netstandard2.0 everywhere else. No clue why they choosed to do that that way.

Reply:
what happens if net46 isn't present on windows? that could be a major issue.

Reply:
I'll check in a sec

Reply:
Ok I am back. it is possible to go netstandard2.0 only by using cake.coreclr.